source/includes/gcp/_images.md

```diff
@@ -0,0 +1,174 @@
+### Images
+
+Images are virtual machine images that have a virtual disk that contains a bootable operating system.
```

Comment:
... that have a virtual disk **_with_** a bootable operating system.?

Reply:
Rephrased.